Add HelmChart not ready to FAQ (#821)
fluxcd/flux2: docs/faq/index.md

```sh
cd ./deploy/prod && kustomize create --autodetect --recursive
```

## Helm questions

### How to debug "not ready" errors?

Misconfiguring the `HelmRelease.spec.chart`, like a typo in the chart name, version or chart source URL
would result in a "HelmChart is not ready" error displayed by:

```console
$ flux get helmreleases --all-namespaces
NAMESPACE	NAME   	READY	MESSAGE
default  	podinfo	False 	HelmChart 'default/default-podinfo' is not ready
```

In order to get to the root cause, first make sure the source e.g. the `HelmRepository`
is configured properly and has access to the remote `index.yaml`:

```console
$ flux get sources helm --all-namespaces 
NAMESPACE  	NAME   	READY	MESSAGE
default   	podinfo	False	failed to fetch https://stefanprodan.github.io/podinfo2/index.yaml : 404 Not Found
```

> If the source is ok, then the error must be caused by an invalid chart name or a non-existing version:

Reviewer: ```suggestion
If the source is `Ready`, then the error must be caused by the chart, for example due to an invalid chart name or non-existing version:
```
